src/main/java/apoc/meta/Meta.java

```diff
@@ -234,12 +237,18 @@ public void rel(int typeId, String typeName, long count) {
             }
 
             public void rel(int typeId, String typeName, int labelId, String labelName, long out, long in) {
-                if (out > 0) relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
-                if (in > 0) relStats.put("()-[:" + typeName + "]->(:" + labelName + ")", in);
+                if (out > 0) {
+                    relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
+                    relStatsCount.put(typeName, out);
```

Comment:
The global relationship-count is available separately in the counts-store.

Here you have to sum it up in my understanding.
`relStatsCount.put(typeName, gelStatsCount.getOrDefault(typeName,0) + out);`